test/ECRecovery.test.js

```diff
@@ -23,32 +28,34 @@ contract('ECRecovery', function (accounts) {
     let message = '0x7dbaf558b0a1a5dc7a67202117ab143c1d8605a983e4a743bc06fcc03162dc0d'; // web3.sha3('OpenZeppelin')
     // eslint-disable-next-line max-len
     let signature = '0x331fe75a821c982f9127538858900d87d3ec1f9f737338ad67cad133fa48feff48e6fa0c18abc62e42820f05943e47af3e9fbe306ce74d64094bdf1691ee53e001';
-    assert.equal(signer, await ecrecovery.recover(message, signature));
+    await ecrecovery.recover(message, signature);
+    assert.equal(signer, await ecrecovery.addrRecovered());
   });
 
   it('recover using web3.eth.sign()', async function () {
     // Create the signature using account[0]
-    const signature = web3.eth.sign(web3.eth.accounts[0], web3.sha3('OpenZeppelin'));
+    const signature = web3.eth.sign(accounts[0], web3.sha3('OpenZeppelin'));
 
     // Recover the signer address form the generated message and signature.
```

Comment:
nitpick: can we update these typos while we're at it? namely `form` to `from` both here and in other places in this file. I know that's annoying, but it'd be appreciated :D